pilot/pkg/model/authentication.go

```diff
@@ -105,6 +105,8 @@ func initAuthenticationPolicies(env *Environment) (*AuthenticationPolicies, erro
 
 func (policy *AuthenticationPolicies) addRequestAuthentication(configs []Config) {
 	for _, config := range configs {
+		reqPolicy := config.Spec.(*v1beta1.RequestAuthentication)
```

Comment:
nit: add a comment about why we need these.

Reply:
done